src/main/js/Navigation.js

```diff
@@ -44,7 +44,7 @@ class Navigation extends Component {
         const components = this.createNavItemForDescriptors(DescriptorUtilities.DESCRIPTOR_TYPE.COMPONENT, DescriptorUtilities.CONTEXT_TYPE.GLOBAL, '/alert/components/');
 
         const nav = (
-            <Fragment>
+            <>
```

Comment:
Interesting that no tag name is needed here.

Reply:
Yes. I looked into this and the "shorthand notation" is preferred. `<>` is equivalent to `<Fragment>`.